sdk/cosmos/azure-cosmos/azure/cosmos/aio/container.py

```diff
@@ -115,27 +114,28 @@ def _set_partition_key(self, partition_key) -> Union[str, Awaitable]:
     @distributed_trace_async
     async def read(
             self,
-            populate_partition_key_range_statistics=None,  # type: Optional[bool]
-            populate_quota_info=None,  # type: Optional[bool]
-            **kwargs  # type: Any
+            **kwargs: Any
     ) -> Dict[str, Any]:
```

Comment:
isn't a breaking change a change that causes a difference in behavior experienced by the end user? I don't believe any of these changes would do that

Reply:
That is a behavioral breaking change, but anything that forces customers to change their code is the classical breaking change... and this is definitely that.

Reply:
Well, it's not _breaking_ because aio was never GA.
If we did it on the sync client it would be breaking - but there are ways we can guide users towards this in a non-breaking way for sync, while fully embracing keyword-only for async.